### PR TITLE
APP-750 JIRA restricted comments

### PR DESCRIPTION
**Pull request by ecarrenho-daitan**
Adding logic and test cases to handle restricted comments.

Comment added/edited events containing restricted comments will be ignored and not pushed as an event into Symphony IM and Chat rooms.

### CODE REVIEW

File: src/test/java/org/symphonyoss/integration/webhook/jira/parser/CommentJiraParserTest.java [hunk 1]
```diff
@@ -90,6 +96,20 @@ public void testParseCommentAdded() throws WebHookParseException, IOException {
   }
 
   @Test
+  public void testParseCommentAddedRestrictedComment() throws WebHookParseException, IOException {
+    ClassLoader classLoader = getClass().getClassLoader();
+    JsonNode node = JsonUtils.readTree(classLoader.getResourceAsStream(FILENAME_COMMENT_ADDED_RESTRICTED_COMMENT));
+    Assert.assertEquals(null, this.commentJiraParser.parse(null, node));
```
Comment (robson-sanchez) [inline src/test/java/org/symphonyoss/integration/webhook/jira/parser/CommentJiraParserTest.java]: You could use the static import.

**Review by robson-sanchez — APPROVED** [review]
LGTM

**Comment by robson-sanchez** [comment]
@dnsymphony please review.

**Review by dnsymphony — APPROVED** [review]

File: src/test/java/org/symphonyoss/integration/webhook/jira/parser/CommentJiraParserTest.java (hunk 1 above)
Comment (ecarrenho-daitan) [reply src/test/java/org/symphonyoss/integration/webhook/jira/parser/CommentJiraParserTest.java]: Done.

**Pull request merged**